### PR TITLE
Add cluster-name flag details and update screenshots

### PR DESCRIPTION
**Pull request by mugdha-adhav**

### CODE REVIEW

File: content/usage/_index.md [hunk 1]
```diff
@@ -11,18 +11,18 @@ Run **@BotKube help** in the Slack channel or directly to the BotKube user to fi
...
 ```
-@BotKube <kubectl command without `kubectl` prefix>
+@BotKube <kubectl command without `kubectl` prefix> [--cluster-name <cluster_name>]
 ```
 See [Examples](/examples/#h-examples) for the use cases.
 
 <h3 class="section-head" id="h-manage-notif"><a href="#h-manage-notif">Managing Slack notifications</a></h3>
 Depending upon your configuration, you will receive Slack notifications about Kubernetes resources lifecycle events and their health.
-BotKube bot allows you to enable/disable notifications using direct messages. Run **@BotKube help** in the Slack channel where the bot is added or as a direct message to the BotKube, the bot will reply with the help message about the supported message formats.
+BotKube bot allows you to enable/disable notifications only in the configured channel. Run **@BotKube help** in the Slack channel where the bot is added or as a direct message to the BotKube, the bot will reply with the help message about the supported message formats.
```
Comment (ssudake21) [inline content/usage/_index.md]: notifications only `from`

File: content/usage/_index.md [hunk 2]
```diff
@@ -39,27 +39,34 @@ This command will open configmap specs in vim editor. Do the required changes, s
...
+Run **@BotKube ping** to the Slack channel where BotKube is added. The BotKube will respond you with the **PONG** message from all the configured clusters. Use `--cluster-name` flag to get response from the cluster mentioned in the flag. Else check the deployment in Kubernetes cluster in the **botkube** namespace.
 
 ![ping](/images/ping.png)
 
 
 <h3 class="section-head" id="h-notifer-stop"><a href="#h-notifier-stop">Disable Slack notifications</a></h3>
 If you want to stop receiving Slack notifications from BotKube, run
 **@BotKube notifier stop**
-to the Slack channel where BotKube is added. You will no longer receive notifications from the BotKube
+in the configured Slack channel where BotKube is added. You will no longer receive notifications from the BotKube
```
Comment (ssudake21) [inline content/usage/_index.md]: from

File: content/usage/_index.md (hunk 1 above)
Comment (mugdha-adhav) [reply content/usage/_index.md]: Made requested changes. Updated in the next commit.

File: content/usage/_index.md (hunk 2 above)
Comment (mugdha-adhav) [reply content/usage/_index.md]: Made requested changes. Updated in the next commit.

**Pull request merged**